bio/seqtk/mergepe/meta.yaml

```
@@ -0,0 +1,19 @@
+name: seqtk mergepe
+description: Interleave two paired-end FASTA/Q files
+url: https://github.com/lh3/seqtk
+authors:
+  - Michael Hall
+input:
+  - paired fastq files - can be compressed.
```

I guess this `compressed` here means gzipped, right? So maybe:
```suggestion
  - paired fastq files - can be compressed in gzip format (``*.gz``).
```